back/app/policies/user_policy.rb

```diff
@@ -8,16 +8,28 @@ def initialize(user, scope)
     end
 
     def resolve
-      scope.all
+      if user&.admin?
+        scope.all
+      elsif user && !user.normal_user?
```

Comment:
I'm confused as to how this is ever true. The user model defines a `normal_user` as `!admin?`, so isn't this just an admin, which would always be covered by the first case: `user&.admin?`. I feel like I am missing something basic here.

Reply:
Yes indeed, but the engines expand this logic to also return false for project and folder moderators. So this branch is about catching the moderators but without hard dependencies from the core to the concepts of project and folder moderators.

Reply:
Perhaps I could add a comment there to make it more clear?